docs/en/users/08_Updating.md

```diff
@@ -0,0 +1,30 @@
+The update process depends on your installation type. 
+If you installes FreshRSS using git, then please refer to our [cli documentation](https://github.com/FreshRSS/FreshRSS/tree/dev/cli).
+
+# Using the web admin panel
+
+*TODO*
+
+# Using git
+
+See [cli documentation](https://github.com/FreshRSS/FreshRSS/tree/dev/cli#install-and-updates).
```

Comment:
Shouldn't this point to master?

Reply:
So, true - it was a bit to late yesterday ;-)